src/app/cli.sh

```diff
@@ -108,7 +108,11 @@ cli::custom() {
 		return 1
 	fi
 
-	drivers::$driver::$sub_cmd || return 1
+	# remove first arg 'custom' from $@ to capture additional args
+	shift
+
+	# pass all remaing args to sub_cmd using $@
+	drivers::$driver::$sub_cmd $@ || return 1
```

Comment:
again, follow the pattern from `main.sh` and quote the arr
https://github.com/benschw/horde/blob/master/src/app/main.sh#L73